drata/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "display_name": "Drata",
+  "maintainer": "support@drata.com",
+  "manifest_version": "1.0.0",
```

Comment:
Hey @Jason-Drata, we're migrating all our integrations to use Manifest version 2.0.0--would you be able to switch to using the manifest 2.0.0 properties? Please see our Manifest reference doc here: https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file